main.go

```diff
@@ -270,6 +272,19 @@ func Main(ctx context.Context, reader InputReader) error {
 
 	ctx = initOtelContext(ctx)
 
+	// add additional attributes if provided to the runtime attributes
+	if addAttributes != "" {
```

Comment:
block: let's add a test a for this.

nit: let's rename `addAttributes` to `additionalAttributes` or `additionalAttrs`, because `add` could cause confusion with _added_ attributes.

nit: we can extract this branch to a testable function, such as `contributeAdditionalAttributes`.

Reply:
What do you think about merging this PR and the other one (one any feedback is addressed). At that point, I will move `main()` under `./cmd/junit2oltp/`, update CI, and move the logic to packages under `./pkg/` and and introduce a struct that holds the configuration and remove the `init()` function. This will abstract thing enough to allow us to implement the tests we are wanting. Currently, there is a cut-line or two missing and it's making it hard to implement the test.

Reply:
Do you think we need a `pkg` outer dir for exporting certain logic? I'd prefer starting with everything under `internal`, and eventually expose it if and only if it's needed by any client code. But I'm open to your ideas.

Reply:
I can see a case where someone doesn't want to use the binary, but instead import and call the logic with buffers instead of stdout/stdin/stderr. Personally, I default to packages being importable. Only things I specifically don't want them depending on do I put in `internal`.

Reply:
Ok. Works for me.